Automation/functionappResourceCreationScript/deploy_template_functionapp.ps1

```diff
@@ -0,0 +1,63 @@
+#parameters required to run the script.
+param(
+    [Parameter(Mandatory = $True)]
+    [string]
+    $subscriptionId,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $tenantId,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $servicePrincipalId,
+
+    [Parameter(Mandatory = $True)]
+    [String]
+    $servicePrincipalPassword,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $resourceGroupName,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $functionAppName,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $storageAccountName,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $appServicePlanName,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $appInsightName,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $location,
+
+    [Parameter(Mandatory = $True)]
+    [string]
+    $templateFilePath
+)
+
+Clear-AzContext -Force;
```

Comment:
Oh, perhaps I should have implemented Clear-AzContext as well. It Deletes all users and groups from the global scope without prompting. Nice!